src/commands/disconnectRepo/IDisconnectRepoContext.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ExecuteActivityContext, ISubscriptionActionContext } from "@microsoft/vscode-azext-utils";
+import { AzureSubscription } from "@microsoft/vscode-azureresources-api";
+import { IGitHubContext } from "../../gitHub/IGitHubContext";
+import { ContainerAppModel } from "../../tree/ContainerAppItem";
+
+export interface IDisconnectRepoContext extends ISubscriptionActionContext, IGitHubContext, ExecuteActivityContext {
+    targetContainer: ContainerAppModel;
```

Comment:
This pair of properties gets repeatedly assigned throughout the codebase... probably a good indicator that we need to create a uniform base context to extend from..